src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,10 @@
+---
+title: New Relic Kubernetes components
+tags:
+  - Integrations
+  - Kubernetes integration
+  - Kubernetes components
+metaDescription: "Learn what components are deployed after installing the Kubernetes integration."
+---
+
+The New Relic Kubernetes integration gives you full observability into the health and performance of your cluster, whether you're running Kubernetes on-premises or in the cloud. It gives you visibility into Kubernetes namespaces, deployments, replicasets, nodes, pods, and containers. 
```

Comment:
I've added content in the kubernetes-components file. This content has been copied and modified from the [Changes introduced in the Kubernetes integration version 3](https://docs.newrelic.com/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/changes-since-v3/) page. Check it out and let me know if this is correct or if I should modify anything.